skyportal/handlers/api/classification.py

```diff
@@ -324,6 +324,7 @@ def get(self, obj_id):
         classifications = obj.get_classifications_readable_by(self.current_user)
         for classification in classifications:
             del classification.groups
+            classification.redshift = obj.redshift
```

Comment:
Our data model associates redshifts with Objs, not classifications. Can you return the `obj` as a nested property of the classification, rather than promoting redshift to a property of the classification?

Reply:
Something like this?
`        for classification in classifications:
            del classification.groups
            classification.obj_data = {'redshift':obj.redshift}`